client/rpc-spec-v2/src/chain_head/tests.rs

```diff
@@ -1024,6 +1024,9 @@ async fn follow_prune_best_block() {
 }
 
 #[tokio::test]
```

Comment:
CC @lexnv

Reply:
Fix published at: https://github.com/paritytech/substrate/pull/13632, will make a note to reenable the tests if this PR gets merged first! Thanks!